### ISSUE TITLE
Release prematurely marked as LTS

### ISSUE THREAD

**Issue by richardlau** [nodejs/nodejs.org#7153]
### URL:

https://nodejs.org/en

### Browser Name:

Edge

### Browser Version:

130.0.2849.56 (Official build) (64-bit)

### Operating System:

Windows 11 Enterprise Version 22H2 build 22621.4169

### How to reproduce the issue:

From earlier today, https://nodejs.org/en was incorrectly advertising Node.js 22.10.0 as Long Term Support (LTS):

![image](https://github.com/user-attachments/assets/a7b7fef7-4ae5-4081-baa6-810c62fb3f94)

This was incorrect -- the first LTS release of Node.js 22 is 22.11.0 (which I've just pushed up in the last couple of hours). I imagine this is some incorrect correlation between the release schedule and current date.

FWIW https://nodejs.org/dist/index.json / https://nodejs.org/dist/index.tab correctly indicate `"lts": false` for Node.js 22.10.0 and `"lts": "Jod"` for Node.js 22.11.0.

**Comment by AugustinMauroy** [comment]
It's an error in logic: we base our calculations on the status, which is calculated from the start and end dates, so the release or not.

**Comment by ovflowd** [comment]
I believe the root issue is within https://github.com/cutenode/nodevu cc @bnb

**Comment by richardlau** [comment]
I think @AugustinMauroy is correct, looks like code in this repo which is checking dates, and not the actual release metadata: https://github.com/nodejs/nodejs.org/blob/90afad3ed29f24f8ed849d6d6cb80680c1e1d7c2/apps/site/next-data/generators/releaseData.mjs#L13-L15

**Comment by ovflowd** [comment]
> I think @AugustinMauroy is correct, looks like code in this repo which is checking dates, and not the actual release metadata: https://github.com/nodejs/nodejs.org/blob/90afad3ed29f24f8ed849d6d6cb80680c1e1d7c2/apps/site/next-data/generators/releaseData.mjs#L13-L15

Makes sense, and then we get the first item with LTS marked.

So the issue is that this was triggered before 22.11 actually went out but on the day that supposedly is valid, right?

**Comment by ovflowd** [comment]
Good thing is that since the data gets updated every 10 minutes (I think?), the links are correct already.

**Comment by richardlau** [comment]
> 
> So the issue is that this was triggered before 22.11 actually went out but on the day that supposedly is valid, right?

Correct.

**Comment by richardlau** [comment]
> I believe the root issue is within https://github.com/cutenode/nodevu cc @bnb

There might _also_ be a bug in nodevu as it appears that might also be doing time-based checks, but at least for this repository it would make sense to start with https://github.com/nodejs/nodejs.org/issues/7153#issuecomment-2445522939.

**Comment by aduh95** [comment]
IMO it makes sense to recommend downloading Node.js 22.x on the day it becomes LTS, most users won't actually care what the metadata says, and it's not like 22.11.0 contained any feature not in 22.10.0.

**Comment by wojtekmaj** [comment]
I guess we can close this issue now :D

**Comment by avivkeller** [comment]
The issue is in regards to all future LTS releases, not just 22

**Comment by avivkeller** [comment]
I don't think this is a planned improvement. As @aduh95 said, there are no differences between the version shown and the version that is actually marked "LTS".

Can this be closed?

**Comment by AugustinMauroy** [comment]
> Can this be closed?

For me yes
